docs/reference/remap/errors/650_chained_comparison_operators.cue

```diff
@@ -0,0 +1,40 @@
+package metadata
+
+remap: errors: "650": {
+	title: "Chained comparison operators"
+
+	description: """
+		You've chained multiple comparison operators together in a way that can't result in a valid expression.
+		"""
+
+	rationale: """
+		Comparison operators can only operate on two operands, e.g. `1 != 2`. Chaining them together, as in
+		`1 < 2 < 3`, produces a meaningless non-expression.
+		"""
+
+	resolution: """
+		Use comparison operators only on a left-hand- and a right-hand-side value. You *can* chain comparisons together
+		provided that expressions are properly grouped. Thus, `1 == (1 == 1)` is a valid expression that yields `false`.
```

Comment:
The reason the user has probably gone down this path is because they want to compare three things. I think a more helpful resolution for what they are actually trying to do would be to say:

> Instead of `a == b == c` you need to compare each pair individually and combine the comparisons with the `&&` like so, `a == b && b == c`.